### PR TITLE
update German translation

### PR DESCRIPTION
**Pull request by lsinger**

### CODE REVIEW

File: languages/wordpress-github-sync-de_DE.po [hunk 1]
```diff
@@ -53,172 +53,176 @@ msgstr ""
...
-#: lib/cli.php:49 wp-github-sync.php:191
+#: lib/cli.php:57 wp-github-sync.php:182
 msgid "Starting full export to GitHub."
 msgstr "Starte einen vollen Export zu GitHub."
 
-#: lib/cli.php:52
-msgid "Exporting post ID to GitHub: "
+#: lib/cli.php:62
+msgid "Exporting post ID to GitHub: %d"
 msgstr "Exportiere Post ID zu GitHub: "
```
Comment (mAAdhaTTah) [inline languages/wordpress-github-sync-de_DE.po]: Needs a `%d` added.

**Comment by mAAdhaTTah** [comment]
Thanks for your contribution!

**Comment by mAAdhaTTah** [comment]
This PR has been integrated manually, credited to you, with the `%d` added as needed. I wanted to get this out with the release today. Thanks for your help!

**Comment by lsinger** [comment]
Sorry for the long delay — life got in the way. Thanks for taking matters into your own hands. :smile:

**Pull request closed without merging**